CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ If you want to submit a bug fix or new feature, make sure that all tests are pas
 
 - Keep git commit messages clear and appropriate. Ideally, follow commit conventions described below.
 - Push your changes to your topic branch in your fork of the repo.
-- Submit a pull request from your topic branch to the master branch in the `@devponess/sdk-js` repository.
+- Submit a pull request from your topic branch to the main branch in the `@devponess/sdk-js` repository.
```

Comment:
Could you fix the repository name too, please? It's currently `@devopness/devopness`.

Reply:
okay!